scipy/stats/_distribution_infrastructure.py

```diff
@@ -5219,3 +5216,128 @@ def __rtruediv__(self, other):
 
     def __neg__(self):
         return self * -1
+
+
+class IMFTransformedDistribution(TransformedDistribution):
+    r"""Distribution underlying an injective, monotonic function of a random variable
```

Comment:
```suggestion
    r"""Distribution underlying a strictly increasing function of a random variable
```

Reply:
There's still an argument to `__init__` for whether the transform is increasing or decreasing and the reciprocal implemented here involves a decreasing transform. Was the suggested change above from before you supported the decreasing case and implemented the reciprocal?

I think `MonotonicTransformedDistribution` would work well here.

Reply:
I think it was from before. Sure, `MonotonicTransformedDistribution` is fine with me.